less/index.less

```diff
@@ -55,5 +55,253 @@ table {
+	linear-gradient(90deg, rgba(0,0,0,.07) 50%, transparent 50%),
+	linear-gradient(90deg, rgba(0,0,0,.13) 45%, transparent 50%),
+	linear-gradient(90deg, transparent 30%, rgba(20,20,20,.17) 60%),
+	linear-gradient(90deg, transparent 50%, rgba(40,40,40,.19) 80%)
+	;
+	// this last bit fixes a few lineup problems by making it a tesselating tile.
+	background-size: 13px, 24px, 37px, 70px;
+
+	// "Why'd you dump it directly onto the body tag?"
+	// "Lack of self-respect, mostly. I can clean it up later!"
+}
+
+div.wrapper {
+	border: 1px solid rgba(135,60,22,1);
+	background-color: @scrollHue;
+}
+
+.wrapper {
+	margin: 15px auto;
+	max-width: 880px;
+	padding: 2%;
+	background-image:
+	linear-gradient(90deg, rgba(135,60,22,0.9), rgba(157,120,3,0.4) 1.5%, rgba(157,120,3,0.2) 3.7%, transparent 9%),
+	linear-gradient(270deg, rgba(135,60,22,0.9), rgba(157,120,3,0.4) 1.5%, rgba(157,120,3,0.2) 3.7%, transparent 9%),
+	linear-gradient(0deg, rgba(135,60,22,0.9), rgba(157,120,3,0.4) 15px, rgba(157,120,3,0.2) 37px, transparent 135px),
+	linear-gradient(180deg, rgba(135,60,22,0.9), rgba(157,120,3,0.4) 15px, rgba(157,120,3,0.2) 37px, transparent 135px),
+	radial-gradient(transparent, rgba(157,120,3,0.2) 95%, rgba(157,120,3,0.4) 96.5%, rgba(135,60,22,0.6) 98%, rgba(135,60,22,0.9) 99%);
+	// Obviously I took notes on other fades but this one was entirely assembled by me.
+	// A series of four linear gradients cover every edge with a "burnt" pattern, and then a radial gradient helps round the corners.
+
+	// .textpadding {
+	// 	padding: 5px;
+	// 	padding-left: 0px;
+	// }
+	// unused class, whoops.
+
+
+	// .partyStyles {
+	// 	// include: partyMainStyles, kenSubStyles, bravaSubStyles, oreSubStyles
+	// }
+	section, header, address {
+		margin: 20px 0;
+		padding: 10px;
+	}
+	nav {
+		padding: 5px;
+		.FlexPen;
+		.navLink {
+			max-width: 25%;
+			margin: 2px 0;
+			padding: 8px;
+			text-decoration: none;
+			font-size: 1.92rem;
+			max-width: 100%;
+
+			// Used to make things look nicer on the tablet transition.
+			@media (max-width: 800px) {
+				width: 33%;
+				text-align: center;
+			}
+
+			// This media query was sometimes tweaked to slightly jump the gun because otherwise the flexboxed links wrap, but I fixed it by toning down the link padding, so it's back to 500.
+			@media (max-width: 500px) {
+				margin: 0;
+				width: 100%;
+				text-align: center;
+			}
+		}
+		// Couldn't find another means to do this that didn't use this double-spec.
+		// Based on the great ideas tablet format.
+		.navLink.partyFlair {
+			@media (max-width: 800px) {
+				text-align: center;
+				width: 100%;
+			}
+		}
+
+	}
+
+	h1 {
+		font-size: 4rem;
+		padding: 5px;
+		padding-left: 0px;
+		margin: 5px 0;
+	}
+	h2 {
+		font-size: 3rem;
+		padding: 5px;
+		padding-left: 0px;
+		margin: 3px 0;
+	}
+	h3 {
+		font-size: 2.4rem;
+		padding: 5px;
+		padding-left: 0px;
+		margin: 1px 0;
+	}
+
+	p {
+		font-size: 1.8rem;
+		padding: 5px;
+		padding-left: 0px;
+	}
+
+	.FlexPen {
```

Comment:
How come you have your Mixins down here? The only feedback I can really give is to put a note to explicitly say  "here are the mixins" or something to that effect. Otherwise it could confuse a new programmer who is going through your code and suddenly gets hit with these.